feature/sessions/src/commonMain/kotlin/io/github/droidkaigi/confsched/sessions/grid/TimetableGrid.kt

```diff
@@ -263,6 +263,13 @@ private fun TimetableGrid(
                     }
                 },
             )
+            // This processing is specific to the JVM; on other platforms, no special processing is performed.
+            .enableMouseWheelZoomForDesktop(
+                multiplyVerticalScaleBy = { factor ->
+                    val before = timetableState.scaleState.verticalScale
+                    timetableState.scaleState.updateVerticalScale(before * factor)
+                },
+            )
```

Comment:
@kitakkun 
Following the comments below, I have changed the name to one that clearly indicates it is intended for the JVM.
This should resolve any concerns preventing the review from proceeding.
https://github.com/DroidKaigi/conference-app-2025/pull/554#discussion_r2327052954